python_frank_energie/frank_energie.py

```
@@ -1337,31 +1376,52 @@ async def smart_battery_details(self, device_id: str) -> SmartBatteryDetails:
         response = await self._query(query)
-        mock_response = {'data': {'smartBattery': {'brand': 'SolarEdge', 'capacity': 16, 'id': "cm3mockyl0000tc3nhygweghn", 'settings': {
-            "batteryMode": "IMBALANCE_TRADING",
-            "imbalanceTradingStrategy": "AGGRESSIVE",
-            "selfConsumptionTradingAllowed": True
-        }}, "smartBatterySummary": {
-            'lastKnownStateOfCharge': 72,
-            'lastKnownStatus': 'CHARGE_IMBALANCE',
-            'lastUpdate': '2025-04-20T11:30:00.000Z',
-            'totalResult': 225.01642490011401
-        }}}
+        mock_response = {
```

Ruff (0.12.2)

Local variable `mock_response` is assigned to but never used

Remove assignment to unused variable `mock_response`

(F841)